### PR TITLE
Create nearly-beta-ready.md

### PR DESCRIPTION
**Pull request by TrechNex**

### CODE REVIEW

File: content/posts/nearly-beta-ready.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Nearly Beta-Ready"
+date: 2019-10-08T19:00:00+01:00
+draft: true
+---
+Instead of providing a weekly reminder of our social media and project links across the web, check out our all-new [Contribute](../../contribute/) page!
```
Comment (BrainBlasted) [inline content/posts/nearly-beta-ready.md]: This could just be `/contribute/`

**Comment by TrechNex** [comment]
Indeed @BrainBlasted - I have notoriously bad commit discipline in this repo and rely on squash & merge. I will endeavour to do better in future

File: content/posts/nearly-beta-ready.md (hunk 1 above)
Comment (TrechNex) [reply content/posts/nearly-beta-ready.md]: I'm hoping that link doesn't break. I've had bad experiences so far with Hugo on this :D

Comment (TrechNex) [reply content/posts/nearly-beta-ready.md]: Change applied

**Comment by BrainBlasted** [comment]
Rebase please so that the preview works :)

**Comment by BrainBlasted** [comment]
Once these are all squashed I'll approve :)

File: content/posts/nearly-beta-ready.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+We have succeeded in creating a pair of ZIP files for Windows 7 or newer, with one being for 32-bit systems and the other for 64-bit. These have been provided as a stop-gap measure until we have working MSI installers and portable executables to publish within the next couple of weeks.
+
+## Flatpak improvements
+Mathieu Bridon has successfully removed the Webkit dependency from our flatpak development builds. This makes builds much faster and reduces the load from the job they produced for our [Jenkins CI server](https://jenkins.glimpse-editor.org).
+
+## Website updates
+Bobby Moss has been drafting the release notes for Glimpse 0.1.0 and a future "downloads" page. You can follow their progress here: https://github.com/glimpse-editor/getglimpse-web/pull/48
+
+You can also now type `www.` at the start of our URL and still be redirected to the right place!
```
Comment (TrechNex) [inline content/posts/nearly-beta-ready.md]: @BrainBlasted should I add a note here about the staging work you've done this evening too?

Comment (BrainBlasted) [reply content/posts/nearly-beta-ready.md]: Sure thing :)

**Pull request merged**